### DISCUSSION TITLE
plz help me

### DISCUSSION THREAD

**Discussion by DeweyPointJr** [KadeArchive/Kade-Engine#2285] [Q&A]
Someone please help me. My game crashes every time i even load it. Like I can't even load it. Here is a video of what happens: https://www.youtube.com/watch?v=Nmbp7OwmIV4 After the video ends i get an error that says "null object reference"

**Comment by NK-523** [comment]
"Null object reference" means the game is trying to load something that doesn't exist, first of all, check "GFtitledance.png", make sure it has a .xml.

**Reply by DeweyPointJr** [reply]
It does have a "GFtitledance.xml" file.

**Reply by NK-523** [reply]
Does "kadeenginelogobumpin.png" have one?

**Reply by DeweyPointJr** [reply]
yep :/
